skills/accessibility/SKILL.md

```
@@ -0,0 +1,223 @@
+When a custom element is slotted inside `<ul>` or `<ol>`, use ElementInternals for the role instead of rendering `<li>` in shadow DOM:
+
+```typescript
+this.#internals.role = 'listitem';
+// Render <div> not <li> in shadow DOM
+```
+
+## FACE (Form-Associated Custom Elements)
+
+### Pattern Selection Decision Tree
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)